client/connector.go

```diff
@@ -580,87 +347,88 @@ func (c *connector) fetchAllTables() error {
 	return nil
 }
 
-func (c *connector) fetchEnumDict(tableName, columnName string) ([]string, error) {
-	var enumDictReponse []string
-
-	resp, err := c.httpClient.Get(c.enumDictPath(tableName, columnName))
-
-	err = c.readJSONResponse(resp, err, &enumDictReponse)
+// fetchEnumDicts fetches all enum dictionaries for enum columns.
+func (c *connector) fetchEnumDicts(table metaCom.Table) (map[int]enumDict, map[int]int, error) {
```

Comment:
do we need to call controller? as discussed, in the future, the disabled autoexpand column should be part of the schema. So once the client get the schema, it should contain the enum already, or this is for current temporary solution?

Reply:
this is temporary.

Reply:
this is only current temporary solution. This part needs to be cleaned up once schema including enum is done.